README.md

```diff
@@ -1,7 +1,7 @@
 # Paragon Sample App
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/useparagon/paragon-connect-nextjs-example)
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fuseparagon%2Fparagon-connect-nextjs-example&env=NEXT_PUBLIC_PARAGON_PROJECT_ID,PARAGON_SIGNING_KEY&envDescription=You%20can%20find%20your%20Paragon%20Project%20ID%20in%20the%20Paragon%20Dashboard%2C%20in%20any%20integration's%20Overview%20tab.%20You%20can%20create%20your%20Paragon%20Signing%20Key%20in%20the%20Settings%20tab%20of%20the%20Paragon%20dashboard.&envLink=https%3A%2F%2Fbit.ly%2F3c4MytJ&demo-title=Paragon%20-%20Sample%20App&demo-description=A%20demo%20Next.js%20app%2C%20with%20integrations%20built%20on%20Paragon.&demo-url=https%3A%2F%2Fparagon-demo-live.vercel.app%2F&demo-image=https%3A%2F%2Fuseparagon.notion.site%2Fimage%2Fhttps%253A%252F%252Fs3-us-west-2.amazonaws.com%252Fsecure.notion-static.com%252Fd28afd31-7e3e-4f05-8b2f-a59f8598baac%252FFrame_977.png%3Ftable%3Dblock%26id%3Dd11fb8bc-ad8c-48fa-b0a7-1ea1dff1f526%26spaceId%3D731f846c-b074-4391-8301-e3172493b9f1%26width%3D1870%26userId%3D%26cache%3Dv2)
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fuseparagon%2Fparagon-connect-nextjs-example&env=NEXT_PUBLIC_PARAGON_PROJECT_ID,PARAGON_SIGNING_KEY&envDescription=You%20can%20find%20your%20Paragon%20Project%20ID%20in%20the%20Paragon%20Dashboard%2C%20in%20any%20integration's%20Overview%20tab.%20You%20can%20create%20your%20Paragon%20Signing%20Key%20in%20the%20Settings%20tab%20of%20the%20Paragon%20dashboard.&envLink=https%3A%2F%2Fbit.ly%2F3c4MytJ&demo-title=Paragon%20-%20Sample%20Apps&demo-description=A%20demo%20Next.js%20app%2C%20with%20integrations%20built%20on%20Paragon.&demo-url=https%3A%2F%2Fparagon-demo-live.vercel.app%2F)
```

Comment:
why "Paragon Sample Apps"?
are we not able to use the old thumbnail img either? i sent you a copy on Slack